modules/API/external/external_api.js

```diff
@@ -1,6 +1,6 @@
 import EventEmitter from 'events';
 
-import { urlObjectToString } from '../../../react/features/base/util';
+import { urlObjectToString } from '../../../react/features/base/util/uri.js';
```

Comment:
the extension is not needed, right?

Reply:
No, the `.js` extension is not necessary.